superchain/configs/rehearsal-0-bn/superchain.toml

```diff
@@ -0,0 +1,11 @@
+name = "rehearsal-0-bn"
+protocol_versions_addr = "0x72B9c5a159B0dEE8331a268791aa432619693c06"
+superchain_config_addr = "0x672F11f34b7bE67C13A6FCba819c339bc3B0a585"
+op_contracts_manager_addr = "0x0fcD165e2531C934a186e3a412D58Daa241E0Ad0"
+
+[hardforks]
+
+[l1]
+  chain_id = 11155111
+  public_rpc = "https://ci-sepolia-l1-archive.optimism.io"
```

Comment:
This just uses the rpc url provided to the following command:
```
just sync-staging --l1-rpc-url=$SEPOLIA_RPC_URL
```
Do we want a different one here?

Reply:
I think this url is validated to have the correct chain id, at least. Doesn't seem problematic to me for devnets to have this. wdyt?

Reply:
Yes it is validated in that way. Seems fine to me as well so will leave